Kernel/sync.sh

```diff
@@ -1,33 +1,68 @@
 #!/bin/bash
 
-if [ "$1" = "-f" ]; then
-    rm -vf _fs_contents
-fi
+set -e
+
+die() {
+	echo "die: $@"
+	exit 1
+}
 
 if [ $(id -u) != 0 ]; then
-    echo "This needs to be run as root"
-    exit 1
+	die "this script needs to run as root"
+fi
+
+echo "setting up disk image..."
+if [ ! -f _disk_image ]; then
+	echo "not found; creating a new one"
+  dd if=/dev/zero of=_disk_image bs=1M count=100 || die "couldn't create disk image"
```

Comment:
100MB is too small if you start building stuff in the `Ports` directory. We're gonna need a bigger default, but maybe there could be a command-line option for overriding the disk image size?

Reply:
Ah yep, I think it was 500MB before. I'll put it back to that, and add an option to change it as suggested.